index.css

```diff
@@ -0,0 +1,268 @@
+body {
+  margin: 0;
+}
+/*Creates flex box for main section and sidebar section, places main and sidebar next to each other as a row*/
+#main {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+}
+/*Styling and flexbox for header*/
+#header {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  font-family: "Roboto", sans-serif;
+  color: darkslategray;
+  margin-bottom: 40px;
+}
+
+#features {
+  margin-top: 60px;
+}
+
+h1 {
+  margin-top: 30px;
+  margin-bottom: 0px;
+  font-family: "Roboto", sans-serif;
+  color: orangered;
+  font-size: 50px;
+  text-shadow: lightgray 2px 2px;
+}
+
+h4 {
+  text-align: center;
+  color: darkslategrey;
+  font-family: "Roboto", sans-serif;
+}
+
+hr {
+  width: 53%;
+  margin-bottom: 4px;
+}
+
+#index-top-images {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin: 10px;
+}
+
+#index-top-images img {
+  margin-left: 10px;
+  margin-right: 10px;
+  border: 5px solid grey;
+  border-radius: 5%;
+}
+
+#features-img-captions {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  font-family: Helvetica, Sans-Serif;
+}
+
+#features-img-captions p {
+  margin-left: 70px;
+  margin-right: 70px;
+  margin-top: 20px;
+  margin-bottom: 40px;
+}
+
+#centered-button {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin-bottom: 75px;
+  margin-top: 30px;
+  font-family: Helvetica, Sans-Serif;
+}
+/*Styling for block quote in the center of the page*/
+#block-quote {
+  display: flex;
+  flex-direction: column;
+  height: 250px;
+  width: 100%;
+  background-color: grey;
+  border-top: 2px solid darkgray;
+  border-bottom: 2px solid darkgray;
+  text-align: center;
+  justify-content: center;
+  font-family: Helvetica, Sans-Serif;
+  color: white;
+  margin-bottom: 50px;
+}
+/*Width and margin for article sections, followed by styling for individual articles*/
+#articles {
+  width: 450px;
+  margin-right: 25px;
+}
+
+.article-1 {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  font-family: Helvetica, Sans-Serif;
+  width: 450px;
+  text-align: justify;
+}
```

Comment:
One thing I wanted to add about some of these main sections to think about for upcoming projects - By giving sections hard-set widths like this example code has (line 106) is not the most responsive way to style a web page. Experimenting with alternate more flexible properties like `[ vw, vh, %, rem, etc . . ]` instead of pixels can be useful.